src/core.ts

```diff
@@ -216,22 +202,58 @@ function TrackerCore(schemas: Schemas, sanitizeEvent: SanitizeEventFn): TrackerF
              * @param event The event to send.
              */
             sendEvent: (event: any) => {
-                if (event && event.eventType === 'sessionChange') {
-                    // This event is deprecated
+                let eventType: string;
+                if (!event || !event.eventType) {
+                    // This should never happen, but there's nothing we can do if it does happen, because the URL
+                    // depends on the event type.
+                    // No log message because we don't want to pollute logs of browser.
+                    return;
+                }
+
+                eventType = event.eventType as string;
+
+                // All event types except these are deprecated.
+                if (eventType !== 'autoSearch' && eventType !== 'search' && eventType !== 'viewProduct' && eventType !== 'addToCart' && eventType !== 'removeFromCart' && eventType !== 'order' && eventType !== 'impression') {
+                    // No log message because we don't want to pollute logs of browser.
                     return;
                 }
 
+                let pathSuffix: string;
+                switch (eventType) {
+                    case 'autoSearch':
+                        pathSuffix = '/gbi-event-autoSearch';
+                        break;
+                    case 'viewProduct':
+                        pathSuffix = '/gbi-event-viewProduct';
+                        break;
+                    case 'addToCart':
+                        pathSuffix = '/gbi-event-addToCart';
+                        break;
+                    case 'removeFromCart':
+                        pathSuffix = '/gbi-event-removeFromCart';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/groupby/gb-tracker-client/pullRequest?prid=7996839)
```suggestion
                        pathSuffix = "/gbi-event-removeFromCart";
```